api/src/main/java/org/eclipse/microprofile/health/spi/SPIFactory.java

```diff
@@ -20,22 +20,15 @@
  *
  */
 
-package org.eclipse.microprofile.health;
+package org.eclipse.microprofile.health.spi;
 
-import java.util.Map;
-import java.util.Optional;
+import org.eclipse.microprofile.health.ResponseBuilder;
 
 /**
- * The health procedure response
+ * Created by hbraun on 07.07.17.
  */
-public interface HealthStatus {
+public interface SPIFactory {
```

Comment:
If there is differents SPI contracts, I'd expect to load each through the ServiceLoader with their respective contract interface (instead of grouping them in a single SPIFactory class).
Especially is some contracts are optional and other mandatory